onedal/datatypes/data_conversion.cpp

```diff
@@ -81,7 +81,7 @@ inline dal::homogen_table convert_to_homogen_impl(PyArrayObject *np_data) {
         column_count = static_cast<std::int64_t>(array_size(np_data, 1));
     }
     const auto layout =
-        array_is_behaved_F(np_data) ? dal::data_layout::column_major : dal::data_layout::row_major;
+        array_is_behaved_C(np_data) ? dal::data_layout::row_major : dal::data_layout::column_major;
```

Comment:
its equivalent, right?

Reply:
not completely. it is possible that both array_is_behaved_F(np_data) and array_is_behaved_C(np_data) are True. before my changes column_major layout was chosen in this case but now row_major is chosen which is default layout in oneDAL.